### DISCUSSION TITLE
Project-level configs

### DISCUSSION THREAD

**Discussion by stewarthsoj** [zed-industries/zed#6546] [Feature Requests]
Is it possible to configure Zed at a project-level? I've got two projects I'm working on and both use different formatters. I can't run the formatters on save at the moment.

**Comment by hovsater** [comment]
It's currently not possible, but there's an [open feature request](https://github.com/zed-industries/community/issues/70) for it. Feel free to upvote it. 🙂

**Comment by JosephTLyons** [comment]
We do have project-level overrides now. In fact, we have folder-specific overrides, where you can override things in certain directories within in a single project.

https://zed.dev/docs/configuring-zed#folder-specific-settings

**Reply by CCXLV** [reply]
Link doesn't work?
